boards/arm/sam_e70_xplained/sam_e70_xplained-common.dtsi

```diff
@@ -150,3 +151,14 @@ zephyr_udc0: &usbhs {
 	pinctrl-0 = <&pd24b_ssc_rf &pa22a_ssc_rk &pa10c_ssc_rd
 		     &pb0d_ssc_tf &pb1d_ssc_tk &pb5d_ssc_td>;
 };
+
+&can0 {
+	status = "okay";
+	pinctrl-0 = <&pb3a_can0_rx0 &pb2a_can0_tx0>;
+	bus-speed = <125000>;
+	sjw = <1>;
+	sample-point = <875>;
+	bus-speed-data = <1000000>;
+	sjw-data = <1>;
+	sample-point-data = <875>;
+};
```

Comment:
I suggest moving `bus-speed`, `sjw`, `bus-speed-data`, and `sjw-data` to SoC DTSI since these will likely be identical for most boards.

Reply:
Yes, but these are typically board-specific configurations. I'll keep them the same as the stm32fd.